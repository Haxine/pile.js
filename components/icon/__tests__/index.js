/**
 * @author renmaomin@126.com
 */
import React from 'react';
import { shallow } from 'enzyme';
import Icon from '..';

describe('Button suite', () => {
  it('render a icon', () => {
    const wrapper = shallow(<Icon type="artboardAdd" />);
    expect(wrapper).toMatchSnapshot();
  });
  it('render a big icon', () => {
    const wrapper = shallow(<Icon type="batchExport" size="large" />);
    expect(wrapper).toMatchSnapshot();
  });
});
