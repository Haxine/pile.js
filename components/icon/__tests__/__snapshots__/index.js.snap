// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button suite render a big icon 1`] = `
<Icon
  size="large"
  type="batchExport"
/>
`;

exports[`Button suite render a bigger icon 1`] = `
<Icon
  size="bigger"
  type="batchExport"
/>
`;

exports[`Button suite render a icon 1`] = `
<Icon
  type="artboardAdd"
/>
`;
